cmd/uplink/cmd/common.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd
+
+import (
+	libuplink "storj.io/storj/lib/uplink"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/uplink"
+)
+
+// loadEncryptionAccess loads the encryption key stored in the file pointed by
+// filepath and creates an EncryptionAccess with it.
+func loadEncryptionAccess(filepath string) (libuplink.EncryptionAccess, error) {
+	key, err := uplink.LoadEncryptionKey(filepath)
+	if err != nil {
+		return libuplink.EncryptionAccess{}, err
+	}
+
+	return libuplink.EncryptionAccess{
+		Key: *key,
+	}, nil
+}
+
+// useOrLoadEncryptionAccess creates an encryption key from humanReadableKey
+// when it isn't empty otherwise try to load the key from the file pointed by
+// filepath and creates an EnryptionAccess with it.
+func useOrLoadEncryptionAccess(humanReadableKey string, filepath string) (libuplink.EncryptionAccess, error) {
+	if humanReadableKey != "" {
+		var key storj.Key
+		copy(key[:], humanReadableKey)
```

Comment:
`storj.NewKey` here too?

Reply:
Same reason than above

Reply:
This still needs to be done.

Reply:
I missed yesterday to swap the `copy` by the `NewKey` after we changed.